### ISSUE TITLE
Incorrectly formats string + helper, prettier-ignore not honored

### ISSUE THREAD

**Issue by jtheberge** [ember-tooling/prettier-for-handlebars-vscode#21]
Formats 
```
<span class='hello--{{get-location planet reallyLongTextThatHitsMaxCharLimit}}'>
```
to
```
<span class='hello--
  {get-location
    planet
    reallyLongTextThatHitsMaxCharLimit
  }}'
>
```
which introduces extra spacing and breaks the desired class such as 'hello--world' to 'hello-- world'.

Adding `{{! prettier-ignore }}` above span also isn't honored

**Comment by jelhan** [comment]
This extension uses an old version of Prettier (v1.18.2). The issues you are facing seem to be fixed in latest version (v2.2.1). At least I can not reproduce in [Prettier Playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAzgBwIZQARgA2266AvAOQAWchhEAtA8MAOZwwP1jYwCW0PJmJQOeAE5xsdAJ4AZaKwAqcAB4wlVXgAk+MdAFlsqgMJbxcvgFs9AX1sUAfKgD0WXI5AAaEBEz9odGRQbHFxCAB3AAVQhCCUaQjsGSCfACNxbDAAaw4AZRwwPihWZBhxAFc4HzgrNLgAEwbGuVxWCux2ADEIcStefhLkEGwKmAhvECoYK0IAdSo9OHcwODy4vT4ANz0ZYbBSSeL0OHEYKMzWfuQAM2kTnwArdFUAIUyc-OwrOEtRW-u1RAz1UeWKrEIcAAihUIPAAYQHiAcOITuJhhDrD90T5MOJijA5nwGjAqMgABwABlx4ROc0ymGGeOWpy2cEmAEdYfALn54iN0AxRI1GpNJFy+JILp1rkg7oigScbGVKorwZCYXD2XLAT4YNg0kSSWSkAAmPWZPiEcEmCBWWUgZYAVkmFROSgN8XlSK2VQAklBmrA8mB8f4AIKBvIwGSQhEnexAA).

I would recommend to prettify the templates with the Prettier version installed in your project instead of using this addon. I have written a [blog post how to configure VSCode and Prettier to prettify Ember templates](https://dev.to/jelhan/format-glimmer-templates-with-prettier-ipa).

**Comment by locks** [comment]
This extension is deprecated, so please use Prettier directly!
